test/tracker-component.test.js

```diff
@@ -9,6 +9,286 @@ import { getUserId, getCartId } from '../src/lib/local-storage';
 
 const { sevenDays, storage } = constants;
 
+const addCartWithExtraItems = [
```

Comment:
Instead of creating these.... you could just have 1 items object and then do:
```
const generateItems = (input, result = []) => {
  if (result.length >== input) {
    return result
  }
  result.push({...sampleItem, id: `${sampleItem[0].id}${result.length}`})
  return generateItems(input, result)
}
...

// then do this:
const items15 = generateItems(15)
....
```

This way, you don't have to commit an array of 30 items (makes the codebase really hard to maintain, and your tests difficult to read)

Reply:
ah thanks for the tip, i'll def add that instead of writing the arrays then